Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -339,6 +338,14 @@ open class CarPlayMapViewController: UIViewController {
         }
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        // Whenever `CarPlayMapViewController` appears on a screen - switch camera to the following
+        // mode.
+        navigationMapView.navigationCamera.follow()
```

Comment:
I still cannot prevent some of the side-effects that occur after panning (e.g. map sometimes starts sliding without reasonable explanation by constantly increasing lat/lng values of the camera center). I have a feeling that some data in `NavigationViewportDataSource` gets corrupted, but still unsure. Will try to resolve in a follow-up ticket.